flake.nix

```
@@ -0,0 +1,22 @@
+{
+  description = "A very basic flake";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+    oxalica.url = "github:oxalica/rust-overlay";
```

There are 2 (different) nixpkgs in `flake.lock`

```suggestion
    oxalica.url = "github:oxalica/rust-overlay";
    oxalica.inputs.nixpkgs.follows = "nixpkgs";
```